tests/stix_translation/test_msatp_stix_to_query.py

```diff
@@ -0,0 +1,218 @@
+from stix_shifter.stix_translation import stix_translation
+import unittest
+import re
+
+translation = stix_translation.StixTranslation()
+
+
+def _remove_timestamp_from_query(queries):
```

Comment:
What's the purpose of removing the timestamps for each test? Is it because the current time keeps changing. I guess I can see this when testing the default time ranges, but why is it needed when you specify START STOP qualifiers in the test pattern?

Reply:
This is implemented for default timestamp